Readme.md

```diff
@@ -81,3 +96,14 @@ To do a manual publish, please follow these steps:
 
 This will take about an hour before the update is available.
 
+## About this repository
+
+The `master` branch of this repository contains the VS Code extension for Prisma schema files. Prisma package dependencies are kept up to date with a GitHub Action workflow, that updates them every time a new version of them is released.
+
+There is a stable version `prisma` and an unstable version `prisma-dev`. The stable one is published as ["Prisma" in the VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma), the unstable one as ["Prisma Dev"](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma-dev). An automated publish runs every 5 minutes calling the `check-update.sh` script: 
+- stable channel:
+  - required versions/values in `package.json` are changed and published
+  - changes are not committed
```

Comment:
`scripts/prisma_version_stable` is one file that is committed by the stable CI job. That enables the future runs to know if an extension version is already published for a specific Prisma CLI version.